content/updates/2024-12-10-editorial-board-update.md

```diff
@@ -0,0 +1,44 @@
+---
+author: ["Mariatta"]
+title: "Meeting Minutes: Dec 10, 2024"
+date: "2024-12-10"
+description: "Meeting Minutes from Python Docs Editorial Board: December 10, 2024"
+summary: "Meeting Minutes from Python Docs Editorial Board: December 10, 2024"
+tags: ["minutes"]
+categories: ["minutes"]
+series: ["Meeting Minutes"]
+ShowToc: false
+TocOpen: false
+members: ["Guido van Rossum", "Mariatta", "Ned Batchelder"]
+---
+
+### Agenda
+
+
+* Is what we’re doing fulfilling the purpose of the EB?
+    * We have been very casual 
+    * We can be more proactive instead of waiting to be asked for a decision.
+    * Tables vs two-line summary: [https://discord.com/channels/935215565872693329/935215566334079058/1313842986836037683](https://discord.com/channels/935215565872693329/935215566334079058/1313842986836037683)
+    * Indecisions about docs are blocking contributions
+    * Diataxis says to make plans and decisions, not sweeping changes
+* Log of decisions:
+    * We have the changelog [https://python.github.io/editorial-board/changelog/](https://python.github.io/editorial-board/changelog/)
+    * People might want a one-size-fits-all decisions but 
```

Comment:
Incomplete sentence?

Reply:
It is incomplete, but that is what we have in our notes.